cosmo_tester/test_suites/snapshots/__init__.py

```diff
@@ -678,21 +686,21 @@ def get_plugins_list(manager, tenant=None):
                 item['distribution'],
             )
             for item in manager.client.plugins.list()
-        ]
```

Comment:
This isn't required for this commit is it- it won't fail flake8 and it's just causing noise in the PR.

Reply:
Fixed